### DISCUSSION TITLE
"connect:" field documentation

### DISCUSSION THREAD

**Discussion by alexlavr** [keystonejs/keystone#7478] [Questions] [answered]
When making a Create or Update operation, I need to relate entries. From code examples, I saw "connect" field. Is it in keystonejs documentation? 

I see documentation for the [schema](https://keystonejs.com/docs/guides/relationships), but not using the API to create relationships.

**Comment by raveling** [comment, marked as answer]
@alexlavr thank you for raising this. We could do more in the docs around this area.

In the interim, you could use the GraphQL Playground to explore the GraphQL API.
